Formula/mavsdk.rb

```diff
@@ -31,7 +35,55 @@ class Mavsdk < Formula
 
   uses_from_macos "zlib"
 
+  resource "Jinja2" do
+    url "https://files.pythonhosted.org/packages/7a/0c/23cbcf515b5394e9f59a3e6629f26e1142b92d474ee0725a26aa5a3bcf76/Jinja2-3.0.0.tar.gz"
+    sha256 "ea8d7dd814ce9df6de6a761ec7f1cac98afe305b8cdc4aaae4e114b8d8ce24c5"
+  end
+
+  resource "MarkupSafe" do
+    url "https://files.pythonhosted.org/packages/67/6a/5b3ed5c122e20c33d2562df06faf895a6b91b0a6b96a4626440ffe1d5c8e/MarkupSafe-2.0.0.tar.gz"
+    sha256 "4fae0677f712ee090721d8b17f412f1cbceefbf0dc180fe91bab3232f38b4527"
+  end
+
+  resource "protoc-gen-mavsdk" do
+    url "https://files.pythonhosted.org/packages/da/91/96d4bde10cf9512250290880373fec0e1e068a9cafae96e69e90f45b2616/protoc_gen_mavsdk-1.0.1-py3-none-any.whl"
```

Comment:
Upstream only provides a wheel, so we can't build this from source. It's only a bootstrap artefact, though, so this is still consistent with what we do elsewhere.